docker/checkout.bash

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-#  Copyright (C) 2018-2021 LEIDOS.
+#  Copyright (C) 2018-2023 LEIDOS.
```

Comment:
Copyright here and in other locations can be extended to 2024.

Reply:
updated